### ISSUE TITLE
404 Warning shown in query monitor plugin

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtCamp/rtsocial#88]
@Mukulsingh27 

When rtSocial plugin is activated, query monitor shows warning related to rtcamp blog post, please check where this post is been used in the plugin as its leading to 404. 

<img width="1054" alt="Screenshot 2022-03-03 at 3 18 33 PM" src="https://user-images.githubusercontent.com/41000648/156558680-bca4fff8-93d1-4f68-81c9-cec2c50b4dff.png">

We can remove it from the plugin. 

cc: @thebengalboy

**Comment by Mukulsingh27** [comment]
@pooja-muchandikar 
I'm not able to re-produce it.

**Comment by pooja-muchandikar** [comment]
@Mukulsingh27 
Uploaded screenshot, you need to have query monitor enabled

**Comment by pooja-muchandikar** [comment]
@Mukulsingh27 

I asked QA team to cross-checking issue by trying to reproduce the issue and found out that the issue is unable to reproduce. Even I tried deactivating all plugins on my site and checked again and found out that now I am not able to reproduce  the issue, so closing this issue for now, will re-open if we encounter this issue again.

**Comment by pavanpatil1** [comment]
```
Environment: 
WordPress: v5.9.1
BuddyPress: v10.1.0
rtMedia: v4.6.10
rtSocial: develop branch. 
```

Able to reproduce the above issue. Hence Reopening this issue. 

**Steps to reproduce:** 
Activate the rtSocial plugin and observed the issue. 

**Note:** Debug log and query monitor should be enabled. Try to reproduce the issue on a fresh installation.

**Comment by pavanpatil1** [comment]
```
Environment:
WordPress: v5.9.1
rtSocial: PR #94 
```
Cross-checked the issue. The fix is working fine.

https://user-images.githubusercontent.com/44057535/157800011-a09f543a-477d-4d9c-a052-844bbda979f0.mp4

**Comment by pavanpatil1** [comment]
Fixed and released in v2.2.3
